File: Python/Handphone.py
#kelas handphone untuk simpan data handphone
class Handphone:

    #konstruktor dengan parameter untuk inisialisasi objek
    def __init__(self, id: int, merek: str, jenis: str, harga: str, spesifikasi: str):
        self._id = int(id)                      #id unik handphone
        self._merek = str(merek)                #merek handphone
        self._jenis = str(jenis)                #jenis atau tipe handphone
        self._harga = str(harga)                #harga handphone
        self._spesifikasi = str(spesifikasi)    #spesifikasi handphone

    #Getter Setter
    def getId(self) -> int:
        return self._id
    def setId(self, id: int) -> None:
        self._id = int(id)

    def getMerek(self) -> str:
        return self._merek
    def setMerek(self, merek: str) -> None:
        self._merek = str(merek)

    def getJenis(self) -> str:
        return self._jenis
    def setJenis(self, jenis: str) -> None:
        self._jenis = str(jenis)

    def getHarga(self) -> str:
        return self._harga
    def setHarga(self, harga: str) -> None:
        self._harga = str(harga)

    def getSpesifikasi(self) -> str:
        return self._spesifikasi
    def setSpesifikasi(self, spesifikasi: str) -> None:
        self._spesifikasi = str(spesifikasi)

File: PHP/Handphone.php
<?php

//kelas untuk simpan data handphone
class Handphone{
    //atribut private hanya bisa diakses getter setter
    private int $id;
    private string $merek;
    private string $jenis;
    private string $harga;
    private string $spesifikasi;
    private string $foto;

    //Constructor untuk inisialisasi objek Handphone baru
    public function __construct(int $id, string $merek, string $jenis, string $harga, string $spesifikasi, string $foto){
        $this->id = $id;
        $this->merek = $merek;
        $this->jenis = $jenis;
        $this->harga = $harga;
        $this->spesifikasi = $spesifikasi;
        $this->foto = $foto;
    }

    //Getter Setter
    public function getId(): int{
        return $this->id;
    }
    public function setId(int $id): void{
        $this->id = $id;
    }

    public function getMerek(): string{
        return $this->merek;
    }
    public function setMerek(string $merek): void{
        $this->merek = $merek;
    }

    public function getJenis(): string{
        return $this->jenis;
    }
    public function setJenis(string $jenis): void{
        $this->jenis = $jenis;
    }

    public function getHarga(): string{
        return $this->harga;
    }
    public function setHarga(string $harga): void{
        $this->harga = $harga;
    }

    public function getSpesifikasi(): string{
        return $this->spesifikasi;
    }
    public function setSpesifikasi(string $spesifikasi): void{
        $this->spesifikasi = $spesifikasi;
    }

    public function getFoto(): string{
        return $this->foto;
    }
    public function setFoto(string $foto): void{
        $this->foto = $foto;
    }
}